frontend/game/index.html

```diff
@@ -11,7 +11,7 @@
         <div id="app" class="text-center container">
             <h1 class="display-1">
                 <a href="/">
-                    Troll Card Game
+                    TCG
```

Comment:
I really hope you are ok with this.
It's too NSFW to have **Troll Card Game** on screen when testing something.

Reply:
good idea